### PR TITLE
TASK: remove outdated useless e2e tests

### LINKED ISSUE #3719: 9.0: Re-Enable Tests in Neos.Neos
Since 9.0 there are currently many tests disabled in Neos.Neos 

For one many outdated behat tests, which were temporary renamed to `.__feature`

But also many functional and unit tests. We need to evaluate if those tests should be reenabled / rewritten -> probably as behat tests.

### PR DESCRIPTION
**Pull request by mhsdesign**
Resolves https://github.com/neos/neos-development-collection/issues/3719 partially by removing the tests.

Related https://github.com/neos/flow-development-collection/issues/3169

**Upgrade instructions**

**Review instructions**

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] ~The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)~
- [x] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [x] Reviewer - The first section explains the change briefly for change-logs
- [ ] ~Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions~

### CODE REVIEW

**Review by dlubitz — APPROVED** [review]
Looks good to me by reading

File: Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature [hunk 1]
```diff
@@ -5,7 +5,7 @@ Feature: Accounts / User Entity Monitoring
   Background:
     Given I have an empty history
     Given I have the following "monitorEntities" configuration:
-    """
+    """yaml
```
Comment (kdambekalns) [inline Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature:8]: What's the purpose here?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -325,18 +325,6 @@ jobs:
...
-        run: |
-          FLOW_CONTEXT=Testing/Behat ./flow configuration:show
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationstatus
-
-          FLOW_CONTEXT=Testing/Behat ./flow behat:setup
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:create
-          FLOW_CONTEXT=Testing/Behat ./flow doctrine:migrationversion --add --version all
-          # TODO: RE-ENABLE AFTER FIXING
-          #bin/behat --stop-on-failure -f progress -c Packages/Neos/Neos.Neos/Tests/Behavior/behat.yml.dist --tags ~@browser
-
```
Comment (kdambekalns) [inline .github/workflows/build.yml:339]: Why? This was added at some point, to make sure minimal divergences in DB behaviour don't caused bugs we hardly notice. If you simply want to make the builds faster, I disagree…

File: Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature (hunk 1 above)
Comment (mhsdesign) [reply Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature:8]: provide ide highlighting for the code fence ;)

File: .github/workflows/build.yml (hunk 2 above)
Comment (mhsdesign) [reply .github/workflows/build.yml:339]: the tests are actually disabled currently see `#` and are run via `composer test:behavioral` (at least for mysql).

I dont know about our postgres status but once we implemented the postgres adapter we will run also `test:behavioral` with postgres, and thus the Neos.Neos behat tests.

File: Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature (hunk 1 above)
Comment (kdambekalns) [reply Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature:8]: Ok… and Behat doesn't care? Interesting… TIL.

Comment (mhsdesign) [reply Neos.Neos/Tests/Behavior/Features/NeosHistory/Entities/AccountsUsers.feature:8]: yes this is so cool ^^ and we use it also for the new escr tests :D

File: .github/workflows/build.yml (hunk 2 above)
Comment (mhsdesign) [reply .github/workflows/build.yml:339]: Issue for that followup task https://github.com/neos/neos-development-collection/issues/3855

**Review by kdambekalns — APPROVED** [review]

**Pull request merged**